run_test.sh

```diff
@@ -59,6 +59,7 @@ function run_lint_fix {
     pip3 install yapf -q
   fi
   yapf -r -i -p --style='{based_on_style: google, indent_width: 2}' server/ tools/ -e=*pb2.py
+  yapf -r -i -p --style='{based_on_style: google, indent_width: 2}' nl_server/ tools/ -e=*pb2.py
```

Comment:
can just add nl_server/ to the command option in line 61?

Reply:
Done